tr/cheatsheet-unsupervised-learning.md

```diff
@@ -1,340 +1,340 @@
 **1. Unsupervised Learning cheatsheet**
 
-&#10230;
+&#10230; Gözetimsiz Öğrenme El Kitabı
 
 <br>
 
 **2. Introduction to Unsupervised Learning**
 
-&#10230;
+&#10230; Gözetimsiz Öğrenmeye Giriş
 
 <br>
 
 **3. Motivation ― The goal of unsupervised learning is to find hidden patterns in unlabeled data {x(1),...,x(m)}.**
 
-&#10230;
+&#10230; Motivasyon ― Gözetimsiz öğrenmenin amacı etiketlenmemiş verilerdeki gizli örüntüleri bulmaktır {x (1), ..., x (m)}.
 
 <br>
 
 **4. Jensen's inequality ― Let f be a convex function and X a random variable. We have the following inequality:**
 
-&#10230;
+&#10230; Jensen eşitsizliği - Bir konveks fonksiyon ve X bir rastgele değişken olsun. Aşağıdaki eşitsizliklerimiz:
 
 <br>
 
 **5. Clustering**
 
-&#10230;
+&#10230; Kümeleme
 
 <br>
 
 **6. Expectation-Maximization**
 
-&#10230;
+&#10230; Beklenti-Enbüyütme (Maksimizasyon) 
 
 <br>
 
 **7. Latent variables ― Latent variables are hidden/unobserved variables that make estimation problems difficult, and are often denoted z. Here are the most common settings where there are latent variables:**
 
-&#10230;
+&#10230; Gizli değişkenler - Gizli değişkenler, tahmin problemlerini zorlaştıran ve çoğunlukla z olarak adlandırılan gizli / gözlemlenmemiş değişkenlerdir. Gizli değişkenlerin bulunduğu en yaygın ortamlar şunlardır:
 
 <br>
 
 **8. [Setting, Latent variable z, Comments]**
 
-&#10230;
+&#10230; Ortam, Gizli değişken z, Yorumlar
 
 <br>
 
 **9. [Mixture of k Gaussians, Factor analysis]**
 
-&#10230;
+&#10230; [K Gaussianların karışımı, Faktör analizi]
 
 <br>
 
 **10. Algorithm ― The Expectation-Maximization (EM) algorithm gives an efficient method at estimating the parameter θ through maximum likelihood estimation by repeatedly constructing a lower-bound on the likelihood (E-step) and optimizing that lower bound (M-step) as follows:**
 
-&#10230;
+&#10230; Algoritma - Beklenti-Enbüyütme (Maksimizasyon) (BE) algoritması, parametrenin maksimum olabilirlik kestirimiyle, olasılığa (E-adımı) tekrar tekrar bir alt-yapı inşa ederek ve bu alt sınırı (M-adımı) aşağıdaki gibi optimize ederek tahmin etmede etkili bir yöntem sunar:
```

Comment:
Algoritma - Beklenti-Ençoklama (Maksimizasyon) (BE) algoritması, θ parametresinin maksimum olabilirlik kestirimiyle tahmin edilmesinde, olasılığa ard arda alt sınırlar oluşturan (E-adımı)  ve bu alt sınırın (M-adımı) aşağıdaki gibi optimize edildiği etkin bir yöntem sunar: